lib/main_common.pm

```diff
@@ -1824,6 +1821,8 @@ sub load_create_hdd_tests {
     loadtest 'console/integration_services' if is_hyperv;
     loadtest 'console/hostname'              unless is_bridged_networking;
     loadtest 'console/force_scheduled_tasks' unless is_jeos;
+    # setup $serialdev permission and so on
+    loadtest "console/consoletest_setup";
```

Comment:
As @okurz mentioned in previous PR, I believe we need to execute consoletest_setup before in the same order as in extra_tests. My only concern is that we have merged and reverted such a change twice already as it broke some tests. Are all of the identified issues resolved now in comparison to the #5387 ? That's the main reason why I'm really conscious about this change.

Reply:
But if we don't do this. There would appear some redundancies to test. For example, #5494. To be honest, I can add "x11/x11_setup" before each "window_system", but there are "x11_setup" already exists in some cases before "window_system" in sle. 

So, I think it's better to merge. How about yours idea?

Reply:
I would rather do small safe changes, so let's aim x11_setup for later. But what's about moving consoletes_setup to the line 1820?

Reply:
#5230 "We should really keep the order of running force_cron_run first to prevent consoletest_setup failing because of performance issues that consoletest_setup wants to prevent." reference from @okurz

Reply:
Hmm, seems that hostname and integration_services do not require consoletest_setup... Ok, let's wait for more reviews if any and try it out. I cannot foresee any issues, but that doesn't mean that there are non =)

Reply:
Ok:)